graphmodel/condition.json

```diff
@@ -35,56 +35,14 @@
             },
             "https://dartnet.info/graphModel/Condition/source": {
                 "@type": ["Object","Code"],
-                "attributes": {
-                    "https://dartnet.info/graphModel/Condition/source/code": {
-                        "@type": "Value",
-                        "attributeName": "code",
-                        "ingestAs": "property"
-                    },
-                    "https://dartnet.info/graphModel/Condition/source/system": {
-                        "@type": "Value",
-                        "attributeName": "system",
-                        "ingestAs": "property"
-                    },
-                    "https://dartnet.info/graphModel/Condition/source/text": {
-                        "@type": "Value",
-                        "attributeName": "text",
-                        "ingestAs": "property"
-                    }
-                }
+                "include":"sourcevalue.schema.json"
             },
             "https://dartnet.info/graphModel/Condition/omop": {
                 "@type": "Array",
                 "arrayElements": {
                     "@id": "https://dartnet.info/graphmodel/Condition/omop/items",
-                    "@type": ["Object","Code","ConditionCode","OMOPConcept"],
-                    "attributes": {
-                        "https://dartnet.info/graphModel/Condition/omop/id": {
-                            "@type": "Value",
-                            "attributeName": "id",
-                            "ingestAs": "property"
-                        },
-                        "https://dartnet.info/graphModel/Condition/omop/name": {
-                            "@type": "Value",
-                            "attributeName": "name",
-                            "ingestAs": "property"
-                        },
-                        "https://dartnet.info/graphModel/Condition/omop/domainId": {
-                            "@type": "Value",
-                            "attributeName": "domainId",
-                            "ingestAs": "property"
-                        },
-                        "https://dartnet.info/graphModel/Condition/omop/vocabularyId": {
-                            "@type": "Value",
-                            "attributeName": "vocabularyId",
-                            "ingestAs": "property"
-                        },
-                        "https://dartnet.info/graphModel/Condition/omop/classId": {
-                            "@type": "Value",
-                            "attributeName": "classId",
-                            "ingestAs": "property"
-                        }
-                    }
+                    "@type": ["Reference", "Code", "ConditionCode"],
+                    "include":"omop-concept.schema.json"
```

Comment:
Put omop concept into the bundle.yaml, and refer to its id here.

Not include, "ref". All omops should be refs